index.html

```diff
@@ -2347,7 +2429,9 @@ <h2>
       </h2>
       <pre class="idl">
         dictionary ImageResource {
-          required USVString src;
+          USVString foreground;
+          USVString background;
+          USVString src;          
```

Comment:
src must be required.

Reply:
That will break some cases though, like if you want to have one entry for a "any" scalable background only - in that case you don't want to require src

Reply:
I have other use cases where we need src. 

Consider, if foreground is missing, “src” is used as foreground. If foreground is present it wins. 

If the above holds, what’s the use case for not requiring src?

Reply:
I had example like this, where background is inferred. When I discussed the feature with some devs, I got a request for that. Background is often very simple and not pixel optimized, so people like using a SVG for that.

```
+                {
 +                  "foreground": "icon/fg.png",
 +                  "sizes": "108x108",
 +                  "type": "image/png"
 +                },
 +                {
 +                  "background": "icon/shared-bg.svg",
 +                  "sizes": "any",
 +                  "type": "image/svg"
 +                }
 +              ]
```